cassovary-core/src/main/scala/com/twitter/cassovary/util/io/GraphReader.scala

```diff
@@ -62,7 +62,7 @@ trait GraphReader[T] {
    * Create an `ArrayBasedDirectedGraph`
    */
   def toArrayBasedDirectedGraph() = {
-    ArrayBasedDirectedGraph(iteratorSeq, parallelismLimit, storedGraphDir)
+    ArrayBasedDirectedGraph(iteratorSeq.map(iter => {() => iter.iterator}).toSeq, parallelismLimit, storedGraphDir)
```

Comment:
Here, `iter.iterator` is returning `empty`. I don't know why it is returning empty. It should return `Iterator[NodeIdEdgesMaxIs]`